AttitudeManager/Src/AM.cpp

```diff
@@ -2,19 +2,34 @@
 // Created by Anthony Luo on 2022-10-12.
 //
 #include "AM.hpp"
+#include "config.hpp"
 
 #include <array>
 #include <cstdlib>
 
+
+
 namespace AM {
 
+AttitudeManager::AttitudeManager(Flightmode* control_algorithm):
+    control_algorithm_(control_algorithm)
+{
+    for (uint8_t i{0}; i < config::NUM_MOTORS; i++) {
+        motorChannels_[i] = config::motors[i].driverConstructor();   
+    }
+};
+
 void AttitudeManager::runControlLoopIteration(const AttitudeManagerInput &instructions) {
     // Process Instructions
 
     // Run Control Algorithms
-    control_algorithm->run();
+    control_algorithm_->run();
 
     // Write motor outputs
 }
 
+void AttitudeManager::outputToMotor(uint8_t percent, uint8_t motorIndex) {
```

Comment:
I think `motorIndex` isn't a great way to output to a particular motor. Could we consider at a different function to handle this? (probably one that uses some of the other config info that is available)